### PR TITLE
Disable persistent cache on arm64 linux

### PR DESCRIPTION
**Pull request by msullivan**

### CODE REVIEW

File: edb/common/debug.py [hunk 1]
```diff
@@ -176,7 +178,15 @@ class flags(metaclass=FlagsMeta):
 
     zombodb = Flag(doc="Enabled zombodb and disables postgres FTS")
 
-    disable_persistent_cache = Flag(doc="Don't use persistent cache")
+    disable_persistent_cache = Flag(
```
Comment (fantix) [inline edb/common/debug.py:181]: Do we want to again flip this to `persistent_cache` now that we allow `default=True` and overriding to `False` to save writing the `DISABLE_` prefix in env var?

**Review by fantix — APPROVED** [review]

File: edb/common/debug.py (hunk 1 above)
Comment (msullivan) [reply edb/common/debug.py:181]: We could, though similar things are mostly `DISABLE` currently

Comment (msullivan) [reply edb/common/debug.py:181]: Yeah, I think I'll stick with DISABLE for nwo

Comment (fantix) [reply edb/common/debug.py:181]: Oh true, yeah DISABLE then.

**Pull request merged**